docker-compose/marklogic-cluster-centos.yml

```diff
@@ -42,7 +42,7 @@ services:
       networks:
       - external_net
     node3:
-      image: store/marklogicdb/marklogic-server:10.0-7.3-centos-1.0.0-ea
+      image: store/marklogicdb/marklogic-server:10.0-8.1-centos-1.0.0-ea2
```

Comment:
We will update this to 10.0-8.3, right?

Reply:
Yeah I just updated it: aeecdd9

Good catch.